public/assets/js/dubtrack/src/views/base/user_popover.view.js

```diff
@@ -36,7 +41,36 @@ dt.global.userPopover = Backbone.View.extend({
 				if(Dubtrack.helpers.isDubtrackAdmin(Dubtrack.session.id) || Dubtrack.room.users.getIfHasRole(Dubtrack.session.id) || Dubtrack.room.model.get('userid') == Dubtrack.session.id) {
 					 this.$('.actions').show();
 					 this.$('.actions a.send-pm-message').show();
-
+					 this.$('.follow-user').show();
+					 this.$('.unfollow-user').hide();
+					 var that = this;
+					 // Check if the user is following the one they are clicking on.. if yes -- show unfollow button, if no .. follow button.
+					 var url = Dubtrack.config.apiUrl + Dubtrack.config.urls.userFollowing.replace( ":id", this.user.get('_id') );
+					 var isFollowing;
+					 $.ajax({
+					 	url: url,
+					 	data: {},
+					 	success: function(data){
+					 	
+					 		$.each(data,function(index, item) {
+					 			
+					 			var followers = _(item).toArray();
+
+					 			$.each(followers, function(index, follower) {
+					 				if(follower.userid == Dubtrack.session.id)
+					 				{
+					 					that.$('.follow-user').hide();
```

Comment:
Here is where you should set that isFollowing, then check isFollowing after the loop to show/hide the follow/unfollow buttons